src/document/json/array.ts

```diff
@@ -17,10 +17,11 @@
 import { TimeTicket } from '../time/ticket';
 import { JSONContainer, JSONElement } from './element';
 import { RGATreeList } from './rga_tree_list';
-
+import { JSONObject } from './object';
 /**
  * JSONArray represents JSON array data structure including logical clock.
  */
+
 export class JSONArray extends JSONContainer {
```

Comment:
It would look good if new lines were added below.

```suggestion
import { JSONObject } from './object';

/**
 * JSONArray represents JSON array data structure including logical clock.
 */
export class JSONArray extends JSONContainer {
```